hs-bindgen-libclang/src/HsBindgen/Clang/Core.hs

```diff
@@ -258,6 +281,16 @@ clang_parseTranslationUnit cIdx src args options =
     withCStrings args $ \args' numArgs -> ensureNotNull $
       nowrapper_parseTranslationUnit cIdx src' args' numArgs mkNullPtr 0 options
 
+-- | Get the normalized target triple as a string.
+--
+-- May throw 'GetTripleFailed'.
```

Comment:
Agreed on both counts, will fix.

Reply:
I'm actually curious if we should be _checking_ these preconditions rather than state them; I think we should, just in case we mess up the bindings. I will change that, also.